app/views/posts/show.html.erb

```diff
@@ -1,23 +1,26 @@
-<% stylesheet_link_tag "tables"%>
+<% stylesheet_link_tag "posts-list"%>
 
+<main class='main-container'>
 <h1>Specific Post</h1>
+<section class='post-extensive'>
+  <div class='post-container'>
+    <div class='post-header'>
+      <h3 class='post-title'><%= @post.title %> (Post #<%= @post.id %>)</h3>
+      <span class='likes-and-comments'>
+        Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %>
+      </span>
+    </div>
+    <p class='post-text'>
+      <%= @post.text %>
+    </p>
+  </div>
+  <div class='post-list-comments'>
+    <% @comments.each do |comment| %>
+      <%= render partial: 'shared/comment-container', locals: { comment: comment, user: @user } %>
+    <% end %>
+  </div>
```

Comment:
- [ ] As requested by the previous CR the post-show page should show all the comments on the post, currently, I can see 5 of them.

| Wureframe | Your design |
|---|---|
| ![image](https://github.com/dev89s/blog-app/assets/58642949/19be83f4-75cf-4011-a326-814d1e9691ab) | ![image](https://github.com/dev89s/blog-app/assets/58642949/04de387e-a5cf-4d51-8fc3-af79dc6ef216) |